### ISSUE TITLE
uncomparables.misc false alarms on "July-August"

### ISSUE THREAD

**Issue by suchow** [amperser/proselint#574]
```
D. Verdon and G. McGraw, "Risk analysis in software design," in IEEE Security & Privacy, vol. 2, no. 4, pp. 79-84, July-Aug. 2004.
```

**Comment by Nytelife26** [comment]
As it stands, both of these are flagged by `misc.phrasal_adjectives.ly`.
